Lib/ctypes/util.py

```diff
@@ -67,6 +67,75 @@ def find_library(name):
                 return fname
         return None
 
+    import ctypes
+    from ctypes import wintypes
+
+    # https://learn.microsoft.com/windows/win32/api/psapi/nf-psapi-enumprocessmodules
+    # https://learn.microsoft.com/windows/win32/api/libloaderapi/nf-libloaderapi-getmodulefilenamew
+
+    _kernel32 = ctypes.WinDLL('kernel32', use_last_error=True)
+    _psapi = ctypes.WinDLL('psapi', use_last_error=True)
+
+    _kernel32.GetCurrentProcess.restype = wintypes.HANDLE
+
+    _kernel32.GetModuleFileNameW.restype = wintypes.DWORD
+    _kernel32.GetModuleFileNameW.argtypes = (
+        wintypes.HMODULE,
+        wintypes.LPWSTR,
+        wintypes.DWORD,
+    )
+
+    _psapi.EnumProcessModules.restype = wintypes.BOOL
+    _psapi.EnumProcessModules.argtypes = (
+        wintypes.HANDLE,
+        ctypes.POINTER(wintypes.HMODULE),
+        wintypes.DWORD,
+        wintypes.LPDWORD,
+    )
+
+    def _get_module_filename(hModule: wintypes.HMODULE):
```

Comment:
Do we allow `ctypes` to avoid PEP 8 naming for Windows names?

Reply:
Renamed these variables